src/Datadog.Trace.ClrProfiler.Managed/Integrations/AspNetCoreMvc2Integration.cs

```diff
@@ -123,6 +129,12 @@ public AspNetCoreMvc2Integration(object actionDescriptor, object httpContext)
         {
             AspNetCoreMvc2Integration integration = null;
 
+            if (!Tracer.Instance.Settings.IsIntegrationEnabled(IntegrationName))
```

Comment:
In lieu of your comment on the tracer instance being able to change, does that matter here? I assume the settings are the same regardless?

Reply:
Yes, it matters here because each `Tracer` instance can have different settings, but it's a bigger change here than simply adding a local variable. We'll revisit this one when we update this integration to follow the same pattern as the others.

To be clear, this is a very unlikely edge case where the user replaces the global `Tracer` while automatic instrumentation is happening (instead of setting it only once during application initialization).